examples/models/llama2/export_llama_lib.py

```diff
@@ -414,6 +416,10 @@ def get_quantizer_and_quant_params(args):
             args.pt2e_quantize, args.quantization_mode
         )
         quantizers.append(qnn_quantizer)
+    if args.coreml and args.pt2e_quantize:
+        assert len(quantizers) == 0, "Should not enable both xnnpack and qnn"
```

Comment:
Is the comment correct `"Should not enable both xnnpack and qnn"`?

Reply:
Yeah seems to be `"Should not enable both xnnpack and coreml"`?